assembl/static2/js/app/components/debate/thread/post.jsx

```diff
@@ -35,12 +35,13 @@ const getFullLevelString = (fullLevel) => {
 // We only have french and english for en, fr, ja for now.
 
 class Post extends React.PureComponent {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
       showAnswerForm: false,
       mode: 'view',
-      showOriginal: false
+      showOriginal: false,
+      isTopPost: !props.data.post.parentId
```

Comment:
there is no need to set that is the state

Reply:
This is true. Can be removed. Use the `this.props.data.post.parentId` further below.